src/app/core/services/issue.service.ts

```diff
@@ -224,16 +224,16 @@ export class IssueService {
       case 'FILTER_BY_TEAM': // Only student has this filter
         const studentTeam = this.userService.currentUser.team.id.split('-');
         filters.push({
-          labels: [this.createLabel('tutorial', studentTeam[0]), this.createLabel('team', studentTeam[1])]
+          labels: [this.createLabel('tutorial', `${studentTeam[0]}-${studentTeam[1]}`), this.createLabel('team', studentTeam[2])]
```

Comment:
If i am not wrong, given `CS2103-W11-1`, this will create a label like `tutorial-CS2103-W11`. Is this how it should be?

Reply:
Expected: `tutorial.CS2103-W11`

Reply:
Hi prof, thanks for the clarification! I have double checked, it outputs `tutorial.CS2103-W11` as expected, instead of `tutorial-CS2103-W11`.